src/coreclr/src/tools/Common/TypeSystem/Common/MethodDesc.cs

```diff
@@ -51,6 +51,14 @@ public sealed partial class MethodSignature : TypeSystemEntity
         // Value of <see cref="EmbeddedSignatureData.index" /> for any custom modifiers on the return type
         public const string IndexOfCustomModifiersOnReturnType = "0.1.1.1";
 
+        // Value of <see cref="EmbeddedSignatureData.index" /> for any custom modifiers on
+        // SomeStruct when SomeStruct *, or SomeStruct & is the type of a parameter or return type
+        // Parameter index 0 represents the return type, and indices 1-n represent the parameters to the signature
+        public static string GetIndexOfCustomModifierOnPointedAtTypeByParameterIndex(int parameterIndex)
+        {
+            return $"0.1.1.2.{(parameterIndex + 1).ToStringInvariant()}.1";
```

Comment:
Not reviewing this part. I don't want to spend time learning how this address is computed because that code upsets me.